code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -1,17 +1,53 @@
-// AI (i.e. game AI, not the AI player) controlled bots
+#define BOT_STEP_DELAY 4 //Delay between movemements
+
+#define DEFAULT_SCAN_RANGE		7	//default view range for finding targets.
+
+//Mode defines
+#define BOT_IDLE 			0	// idle
+#define BOT_HUNT 			1	// found target, hunting
+#define BOT_PREP_ARREST 	2	// at target, preparing to arrest
+#define BOT_ARREST			3	// arresting target
+#define BOT_START_PATROL	4	// start patrol
+#define BOT_PATROL			5	// patrolling
+#define BOT_SUMMON			6	// summoned by PDA
+#define BOT_CLEANING 		7	// cleaning (cleanbots)
+#define BOT_REPAIRING		8	// repairing hull breaches (floorbots)
+#define BOT_MOVING			9	// for clean/floor/med bots, when moving.
+#define BOT_HEALING			10	// healing people (medbots)
+#define BOT_RESPONDING		11	// responding to a call from the AI
+#define BOT_DELIVER			12	// moving to deliver
+#define BOT_GO_HOME			13	// returning to home
+#define BOT_BLOCKED			14	// blocked
+#define BOT_NAV				15	// computing navigation
+#define BOT_WAIT_FOR_NAV	16	// waiting for nav computation
+#define BOT_NO_ROUTE		17	// no destination beacon found (or no route)
 
-/obj/machinery/bot
+// AI (i.e. game AI, not the AI player) controlled bots
+/mob/living/simple_animal/bot
 	icon = 'icons/obj/aibots.dmi'
 	layer = MOB_LAYER
 	luminosity = 3
-	use_power = 0
-	var/obj/item/weapon/card/id/botcard			// the ID card that the bot "holds"
+	stop_automated_movement = 1
+	wander = 0
+	healable = 0
+	damage_coeff = list(BRUTE = 1, BURN = 1, TOX = 0, CLONE = 0, STAMINA = 0, OXY = 0)

```

Comment:
This isn't set right, 1 is for ignored damage types, 0 for allowed types.
Which is stupid, but hey.